app/views/items/index.html.erb

```diff
@@ -10,14 +10,20 @@
       <h2> <%=link_to item.name, "/items/#{item.id}" %> </h2>
       <p>Sold by: <%=link_to item.merchant.name, "/merchants/#{item.merchant.id}" %></p>
       <%= link_to image_tag("#{item.image}"), "/items/#{item.id}" %>
-      <p> <%= item.description unless @merchant%> </p>
+      <p> <%= item.description %></p>
       <p>Price: <%=number_to_currency(item.price) %> </p>
       <p>Inventory: <%= item.inventory %> </p>
       <% if @merchant %>
         <% if item.active? %>
           <p>Active</p>
+          <%= link_to "Deactivate", "/merchant/items/#{item.id}", method: :patch %><br>
         <% else %>
           <p>Inactive</p>
+          <%= link_to "Activate", "/merchant/items/#{item.id}", method: :patch %><br>
+        <% end %>
+
+        <% if item.no_orders? %>
+          <%= link_to "Delete", "/merchant/items/#{item.id}", method: :delete %><br>
         <% end %>
```

Comment:
I understand this now. If a user is not a merchant it won't show them the functionality of activating or deactivating an item. I like this solution better than routing to a new view for a merchant to have functionality over their own items. Disregard my last comment